src/NuGet.Core/NuGet.Packaging/PackageExtractor.cs

```diff
@@ -89,8 +91,13 @@ public static class PackageExtractor
                 throw new ArgumentNullException(nameof(packagePathResolver));
             }
 
-            // TODO: Need to handle PackageSaveMode
-            // TODO: Support overwriting files also?
+            if (packageExtractionContext == null)
+            {
+                packageExtractionContext = new PackageExtractionContext();
```

Comment:
Perhaps, I can change the callers to pass in a non-null PackageExtractionContext always.